### PR TITLE
feat(ui): make color picker movable

### LINKED ISSUE #332: [v1.15.0] Movable Neopixel Color Picker.
## Is your feature request related to a problem? Please describe.
When trying out the new Neopixel Color Picker I noticed that the window that opens is not movable.
This makes it a bit hard to see the changes if your layout has the camera/console above the color picker.

## Describe the solution you'd like:
Simply make the color picker movable, freely movable would be the best, like click and drag it around. 

## Affected users and/or setups:
Would help people who have their fan controls and camera a bit separate. Just a small change.

### PR DESCRIPTION
**Pull request by matmen**
Adds a draggable title to the color picker card - now they can be moved anywhere you'd like ;)
chrome_y97UR5d20e

Closes #332

### CODE REVIEW

**Comment by pedrolamas** [comment]
Just checking, but have you tested this on the phone too?

**Comment by pedrolamas** [comment]
I just tested in Windows + MSEdge and this is not working correctly (mouse pointer is not over title bar and text gets selected)

record

**Comment by matmen** [comment]
Oops, looks like I have some more browser testing to do. Thanks for the QA.

File: src/components/ui/AppColorPicker.vue [hunk 1]
```diff
@@ -154,6 +164,8 @@ export default class AppColorPicker extends Vue {
   dot!: boolean
 
   menu = false
+  dragging = false
+  lastPointerPosition: {x: number, y: number} = { x: 0, y: 0 }
```
Comment (pedrolamas) [inline src/components/ui/AppColorPicker.vue:168]: As you have a couple of places with `{x: number, y: number}`, probably best to just create a local type and reuse that?

**Comment by pedrolamas** [comment]
Just tested with the new changes, seems to be working fine on desktop (haven't tried phone yet!)

**Review by pedrolamas — DISMISSED** [review]

File: src/components/ui/AppColorPicker.vue [hunk 2]
```diff
@@ -262,6 +279,63 @@ export default class AppColorPicker extends Vue {
...
+  stopMouseDrag () {
+    this.dragging = false
+  }
+
+  startTouchDrag (event: TouchEvent) {
+    this.lastPointerPosition = { x: event.touches[0].clientX, y: event.touches[0].clientY }
+  }
+
+  relativeMove (newPosition: PointerPosition) {
+    if (!this.$refs.card) {
```
Comment (pedrolamas) [inline src/components/ui/AppColorPicker.vue:297]: Up to you on this one, but I prefer to use refs like here: https://github.com/fluidd-core/fluidd/blob/4cf6f4348931d86460b5595951466e08867c81de/src/components/ui/AppChart.vue#L39-L40

Comment (matmen) [reply src/components/ui/AppColorPicker.vue:297]: Oh yeah, I like consistency. Will change that.

Comment (pedrolamas) [reply src/components/ui/AppColorPicker.vue:297]: We got both approaches all around the app, so consistency is long gone! 😁

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**